docs/marketplace/user/merchant-portal-user-guides/202204.0/products/concrete-products/managing-marketplace-concrete-product-prices.md

```diff
@@ -18,19 +19,29 @@ This document contains reference information. Make sure to review it before you
 
 ## Adding a marketplace concrete product price
 
-To add a new price for a marketplace concrete product:
+To add a new price for a marketplace concrete product, follow these steps:
 
-1. On the **Concrete Products** page, next to the concrete product you want to edit, hover over the three dots and click **Manage Product** or just click the line. The **Concrete Product SKU, Name** page opens.
+1. On the **Concrete Products** page, next to the concrete product you want to edit, hold the pointer over the three dots and click **Manage Product** or just click the line. The **Concrete Product SKU, Name** page opens.
 2. Scroll down to the **Price** pane.
 3. In the **Price** pane, click **+Add**. The empty cells appear in the following table.
-4. From the drop-down menu **Store**, select the store for which the price is created.
-5. From the drop-down menu **Currency**, select the currency in which the price is defined.
-6. For the **NET DEFAULT** cell, enter a price. Use `.` or `,` separators.
-7. For the **GROSS DEFAULT** cell, enter a price. Use `.` or `,` separators.
-8. Optional: For the **GROSS DEFAULT** cell, enter a price. Use `.` or `,` separators.
-9. Optional: For the **NET ORIGINAL** cell, enter a price. Use `.` or `,` separators.
-10. Optional: For the **GROSS ORIGINAL** cell, enter a price. Use `.` or `,` separators.
-11. For the **QUANTITY** cell, enter the number of items. By default, the quantity is 1. For an example, see [Adding volume prices](#adding-volume-prices).
+4. Optional: To set a price for a specific customer, from the **Customer** drop-down menu, select the appropriate [business unit](/docs.spryker.com/docs/scos/user/features/{{page.version}}/company-account-feature-overview/business-units-overview.html) to apply the price to.
+
+  {% info_block infoBox "Note" %}
+
+  In order for the business unit to which the customer is assigned to appear in the **Customers** drop-down menu, in the Back Office, create the [merchant relation](/docs/scos/user/back-office-user-guides/{{page.version}}/marketplace/merchants-and-merchant-relations/managing-merchant-relations.html#prerequisites).
+
+  You can set customer-specific prices set for [marketplace products](/docs/marketplace/user/features/{{page.version}}/marketplace-product-offer-feature-overview.html), not [product offers](/docs/marketplace/user/features/{{page.version}}/marketplace-product-offer-feature-overview.html).
```

Comment:
also cannot combine, specific prices with volume prices

Reply:
@AdvaBD So do we need to add an extra note—for example, "Also, you cannot combine customer-specific prices with volume prices"

Reply:
yes